build_tools/cirrus/arm_wheel.yml

```diff
@@ -69,3 +69,31 @@ linux_arm64_wheel_task:
 
   wheels_artifacts:
     path: "wheelhouse/*"
+
+
+wheels_upload_task:
+  depends_on:
+    - macos_arm64_wheel
+    - linux_arm64_wheel
+  container:
+    image: continuumio/miniconda3:22.11.1
+  # Artifacts are not uploaded on PRs
+  only_if: $CIRRUS_PR == ""
+  env:
+    # XXX: Need to encrypt keys in the CirrusCI interface
```

Comment:
I don't understand this comment. Does this require manual action before or after this PR is merged to `main` (and potentially backported to `1.2.X`)?